index.html

```diff
@@ -189,111 +224,118 @@ <h4 class="colorh4">CANOPY</h4>
         </article>
     </section>
 
-    <!----jkncvevkefbm,rfbm,rbm, -->
+    <!----about me section -->
     <section id="aboutme">
-        <article class="padding">
-            <h1>About Myself</h1>
-            <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
-                work
-                and experience! If you like what you see and have a project you need coded, don't hesitate to contact
-                me.
-            </p>
-            <h4>LET'S CONNECT</h4>
-            <ul>
-                <li>
-                    <a title="The link will take you to my facebook profile" href="https://www.facebook.com/Mpdaddy/">
-                        <i class="fa fa-facebook social"></i>
-                    </a>
-                </li>
-                <li>
-                    <a title="The link will take you to my twitter profile" href="https://github.com/MPDADDY">
-                        <i class="fa fa-twitter social"></i>
-                    </a>
-                </li>
-                <li>
-                    <a title="The link will take you to my instagram profile"
-                        href="https://www.instagram.com/mpdaddympd/">
-                        <i class="fa fa-instagram social"></i>
-                    </a>
-                </li>
-                <li>
-                    <a title="The link will take you to my linkin profile"
-                        href="https://www.linkedin.com/in/anold-mwawasi">
-                        <i class="fa fa-linkedin social"></i>
-                    </a>
-                </li>
-            </ul>
-            <div class="getresume">
-                <a title="get my resume" href="#">Get my resume</a>
-            </div>
-        </article>
-        <article id="what_i_do" class="padding">
+        <div class="aboutmyself ">
+            <article class="padding">
+                <h1>About Myself</h1>
+                <p>I'm a software developer! I can help you build a product , feature or website Look through some of my
+                    work
+                    and experience! If you like what you see and have a project you need coded, don't hesitate to
+                    contact
+                    me.
```

Comment:
- [ ] Lastly, your `About section` looks great but in order to implement the `About me` content taking up lesser space than the `languages, skills and frameworks` as the previous reviewer suggested, I will suggest you style the main container of the section with` display: grid` and ensure that the `about me` part is wrapped in a separate container and takes 40% of the page and the `languages, skills and frameworks` also wrapped in a separate container and takes 60% of the page.
- This way your design looks more similar to the Figma design 😉 
- Your Design
<img width="1273" alt="Screen Shot 2023-05-12 at 6 20 34 PM" src="https://github.com/MPDADDY/portfolio/assets/102699958/cc521eb8-f970-40c3-afc7-0d5b9fcb63ad">

- The Figma Design

<img width="719" alt="Screen Shot 2023-05-12 at 6 20 22 PM" src="https://github.com/MPDADDY/portfolio/assets/102699958/e446f1d5-f3ef-446b-9616-da120496bd05">